### PR TITLE
Adding contact us page content and updates to main.css and functions within main.js

### LINKED ISSUE #13: Setup "Contact us" HTML file
Start HTML boiler plate for CONTACT US page and include bootstrap script/css tags

# User Story
User will click on CONTACT US in the navbar and will be directed to the CONTACT US page. They will be able to add their name, email address and favorite type of bloomer. Company information will be displayed on the right side of the screen.

# Acceptance Criteria

WHEN the user visits the CONTACT US page
THEN a form will appear with a picture and company information
AND the user will add their name to the form
AND the user will add their email address to the form
AND the user will choose a favorite bloomer from a dropdown menu
AND the user will click a button to submit their information

# Dependecies
#19 
#27 

# Dev Notes
Style navbar and footer before setting up.

### LINKED ISSUE #15: Setup "Contact Us" CSS
Style CONTACT US to the specifications agreed on by the group

# User Story
The user will see a wonderful and beautiful CONTACT US page when they click on the link from the navbar.

# Acceptance Criteria

WHEN the user clicks on the link to CONTACT US
THEN they will see a good looking CONTACT US page

# Dependecies
#13

# Dev Notes
#13

### LINKED ISSUE #25: Setup "Contact Us" JS file
User data will be captured into an array of objects from the FORM when the submit button is clicked. A for loop will choose a random picture to display on the right of the screen.

# User Story
The user will add their name, email and favorite bloomer type in the FORM which will be added into an array.

# Acceptance Criteria

WHEN the user inputs their name, email & favorite bloomer
THEN the data will be stored into an array
AND a for loop will generate a random picture on right side of the screen

# Dependencies
#13 

# Dev Notes
#13

### PR DESCRIPTION
**Pull request by ryanmcnair**
Pushing up all Contact us page HTML with CSS styles and JS functions.

## Description
Adding Contact us page content with styles and functionality

## Related Issue
Closes #15 
Closes #13 
Closes #25 

## Motivation and Context
This will add the Contact us page to the application

## How Can This Be Tested?
My branch can be pulled and use hs -o.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: setup/main.css [hunk 1]
```diff
@@ -1,24 +1,118 @@
 /* UNIVERSAL STYLES */
 
-body {
-  background-color: aquamarine;
-}
 /* NAVBAR */
 .navbar-brand {
   font-family: 'Rye', cursive;
+  background-color: #fcf7f0;
```
Comment (danapham) [inline setup/main.css]: Please change the background color on navbar to white.

File: setup/main.css [hunk 2]
```diff
@@ -1,24 +1,118 @@
...
+#contact-message {
+    display: flex;
+    justify-content: center;
+    margin-top: 10%;
+}
+
+#thanks {
+    height: 30%;
+    width: 45%;
+    font: 'roboto', cursive;
```
Comment (danapham) [inline setup/main.css]: I believe this should font-family.

**Review by danapham — COMMENTED** [review]
Added some notes for you to fix up. Just some small styling changes.

**Review by danapham — APPROVED** [review]
Nice job!

**Review by josephtmartin — APPROVED** [review]
+1

**Pull request merged**